index.html

```diff
@@ -944,14 +944,13 @@ <h2>Dictionary <a class="idlType">DisplayMediaStreamOptions</a>
                   will not contain an audio track.</p>
                 </dd>
                 <dt>
-                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}})</code>,
-                  defaulting to <code>null</code>
+                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}})</code>
```

Comment:
```suggestion
                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}}</code>
```
<img width="333" alt="image" src="https://user-images.githubusercontent.com/634478/193274869-903fe973-80cb-48c1-adbc-13ce0e786ca5.png">